src/main/java/net/greatstart/services/UserService.java

```diff
@@ -52,6 +52,7 @@ public void deleteUser(long id) {
     }
 
     public User getUserById(long id) {
+        System.out.println("return user by id " + id);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)